.gitignore

```diff
@@ -88,3 +88,5 @@ bower_components
 **/generated
 */generated_tests
 /bin/
+ui/PipelineTest.java
+ui/CameraTest.java
```

Comment:
These are actually tests. They were files we had created during the process that were not meant to be put inside of git since they changed every time you used the generator. They are the files that the generator is saving to currently. When we add the feature that allows you to choose where to save the file these lines can be removed since the files won't be generated there.